### PR TITLE
[feat] Wishlist Page added

### PR DESCRIPTION
**Pull request by shivampandey0**
- links updated on homepage and cart page
- Wishlist page added

image

### CODE REVIEW

File: css/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+.card-icons > button {
+  color: var(--primary);
+  opacity: 0.8;
+}
+
+@media screen and (max-width:600px) {
+    .card{
+        width: 60%;
+    }
+}
```
Comment (hinanshisuthar) [inline css/wishlist.css]: Add a new line at the EOF.

**Review by hinanshisuthar — COMMENTED** [review]
Amazing UI, Shivam. Looks awesome. 
Also, update the wishlist count and cart count on the navbar of the homepage.

**Review by saadahmadhere — APPROVED** [review]
Sleek. Nice use of variables. Just take care of the EOF line. Good to merge.

File: css/wishlist.css (hunk 1 above)
Comment (shivampandey0) [reply css/wishlist.css]: Added

**Comment by shivampandey0** [comment]
Refactored code with Review Suggestions.

**Pull request merged**